Index: linux-3.10.23/crypto/Kconfig
===================================================================
--- linux-3.10.23.orig/crypto/Kconfig
+++ linux-3.10.23/crypto/Kconfig
@@ -1406,3 +1406,6 @@ source "drivers/crypto/Kconfig"
 source crypto/asymmetric_keys/Kconfig
 
 endif	# if CRYPTO
+
+source "crypto/ocf/Kconfig"
+
Index: linux-3.10.23/crypto/Makefile
===================================================================
--- linux-3.10.23.orig/crypto/Makefile
+++ linux-3.10.23/crypto/Makefile
@@ -96,6 +96,8 @@ obj-$(CONFIG_CRYPTO_USER_API) += af_alg.
 obj-$(CONFIG_CRYPTO_USER_API_HASH) += algif_hash.o
 obj-$(CONFIG_CRYPTO_USER_API_SKCIPHER) += algif_skcipher.o
 
+obj-$(CONFIG_OCF_OCF) += ocf/
+
 #
 # generic algorithms and the async_tx api
 #
