Index: iptables-1.4.17/extensions/GNUmakefile.in
===================================================================
--- iptables-1.4.17.orig/extensions/GNUmakefile.in
+++ iptables-1.4.17/extensions/GNUmakefile.in
@@ -23,7 +23,7 @@ kinclude_CPPFLAGS  = @kinclude_CPPFLAGS@
 AM_CFLAGS       = ${regular_CFLAGS}
 AM_CPPFLAGS     = ${regular_CPPFLAGS} -I${top_builddir}/include -I${top_builddir} -I${top_srcdir}/include ${kinclude_CPPFLAGS}
 AM_DEPFLAGS     = -Wp,-MMD,$(@D)/.$(@F).d,-MT,$@
-AM_LDFLAGS      = @noundef_LDFLAGS@
+AM_LDFLAGS      = ${noundef_LDFLAGS} -L${top_srcdir}/libiptc/.libs
 
 ifeq (${V},)
 AM_LIBTOOL_SILENT = --silent

